Formula/tiny-fugue.rb

```diff
@@ -14,13 +14,16 @@ class TinyFugue < Formula
   conflicts_with "tee-clc", :because => "both install a `tf` binary"
 
   depends_on "libnet"
+  depends_on "openssl"
   depends_on "pcre"
 
   # pcre deprecated pcre_info. Switch to HB pcre-8.31 and pcre_fullinfo.
   # Not reported upstream; project is in stasis since 2007.
   patch :DATA
 
   def install
+    ENV.append "CPPFLAGS", "-I#{Formula["openssl"].opt_prefix}include"
```

Comment:
Yeah, we have some "magic" in place to help formulae locate `keg_only` dependencies so most of the time things like this aren't necessary, beyond the `depends_on` line.